hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/CompressionCodecFactory.java

```diff
@@ -29,6 +29,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.hadoop.util.StringUtils.toLowerCase;
```

Comment:
Just for this, there is no need to the import, just use StringUtils.toLowerCase() in 204.

Reply:
Removed, and fixed.